CHANGELOG.md

```diff
@@ -1,6 +1,76 @@
 # Changelog
 ## Upcoming
 
+# PrimeHub v3.11.0
+
+### Notable Changes
+
+- Group deletion will include Dataset and SharedFiles
+- Deployment and PhApps are airgap compatible
+
+Example:
+
+```yaml
+controller:
+  airgap:
+    imagePrefix: primehub.airgap:5000/
+```
+
+
+## Dataset 2.0
+
+- Update tipsLink of datasets page (sc-23508)
+- Datasets 2.0 Enhancement: Create/update a dataset from shared files (sc-23403)
+- Datasets/sharedFiles - the content of button will be updated accordingly (sc-19825)
+- Replace the uppy uploader (tusd) by ant uploader (rest) (sc-23401)
+- Update a dataset from shared files - able to add files into directory (sc-23940, sc-23500)
+- [Bugfix] Failed to create/update dataset from SharedFiles (sc-23982)
+
+
+## SDK
+
+- Add Dataset 2.0 support in PrimeHub SDK/CLI (sc-22605)
+- The upload source and destination should follow the copy logic (sc-23727)
+- Add Secrets support in PrimeHub SDK/CLI (sc-23691)
+- [Bugfix] files command not output valid json (sc-23681)
+
+## License
+- Support License by # of user (sc-23693)
+
+
+## PrimeHub Apps
+
+- Upgrade Streamlit to v1.2.0 (sc-24357)
+- PhApp already exists error when importing PhApp template (sc-23484)
+- [Bugfix] The behavior of reload button in Apps is unclear (sc-20463)
+- [Bugfix] Update PhApplication might make API crash (sc-23440)
+
+## Deployment
+
+- [Bugfix] Deployment show error page if deployment is deleted (sc-23539)
```

Comment:
- [Bugfix] An error page is now correctly displayed when trying to view a deleted deployment (sc-23539)